mppm/virtual-players/virtual-players-enable.md

```diff
@@ -1,22 +1,36 @@
 ---
 id: virtual-players-enable
-title: Enable virtual players
+title: Activate a Virtual Player
 description: Enable up to 3 additional Virtual Players to your Main Editor Player in Multiplayer Play Mode.
 ---
-:::important
-You can only attach **Player Tags** to inactive players (including the **Main Editor Player**). If you plan to use **Player Tags**, [create and configure your tags](../player-tags/player-tags-create.md) before you enable virtual players.
+
+To use a Virtual Player to test multiplayer gameplay, activate at least one Virtual Player.
+
```

Comment:
Can we clarify that this folder is only created the first time you launch a virtual player

Reply:
If you delete the folder it will also create a new one, or if you update mppm if should clear these folders as well. Otherwise it will use the folder it has cached.